src/ctap2/mod.rs

```diff
@@ -862,10 +861,13 @@ pub(crate) fn bio_enrollment(
     let mut pin_uv_auth_result = puat_result
         .clone()
         .unwrap_or(PinUvAuthResult::NoAuthRequired);
+    // See, if we have a cached PUAT with matching permissions.
     match puat_result {
         Some(PinUvAuthResult::SuccessGetPinToken(t))
         | Some(PinUvAuthResult::SuccessGetPinUvAuthTokenUsingUvWithPermissions(t))
-        | Some(PinUvAuthResult::SuccessGetPinUvAuthTokenUsingPinWithPermissions(t)) => {
+        | Some(PinUvAuthResult::SuccessGetPinUvAuthTokenUsingPinWithPermissions(t))
+            if t.permissions == PinUvAuthTokenPermission::BioEnrollment =>
```

Comment:
We want to check membership rather than equality. Looks like the version of bitflags that we're using has a `contains` function.

(aside: I'm a little concerned that more recent versions of bitflags are quite different. Long term we should remove that dependency. It's not really doing much for us.)

Reply:
Ok, done. 
We also need to think about adding a `rust-toolchain.toml`-file to the repo, to avoid clippy randomly breaking the CI.
From what I can tell, this time it was a clippy-bug and not a real problem.